cypress/integration/login.js

```diff
@@ -11,14 +11,14 @@ describe('Login functionality', function() {
         cy.get('[id="loginForm.userId"]').click({force:true})
         cy.get('[aria-label="Demo User"]').click()
         cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="User"]').click()
+        cy.get('[aria-label="Team Lead"]').click()
         cy.get('[type="submit"]').click()
-
         cy.url().should('include', '/time-logging')
         cy.get('.page__title').contains('Timesheets')
         cy.get('.calendar').should('be.visible')
         cy.get('.tile.form').should('be.visible')
         cy.get('.user-info__title').contains('Demo User')
-        cy.get('.main-nav').find('li').should('have.length', 1)
+        cy.get('.main-nav').find('li').should('have.length', 2)
+        cy.get('[data-test="timesheets-current-day"]').contains('15')
```

Comment:
1. Is the [data-test="timesheets-current-day"] locator unique?
2. This test will fail tomorrow, maybe we can do better, e.g. https://www.w3schools.com/jsref/jsref_getdate.asp